test/PowerShellEditorServices.Test.E2E/DebugAdapterClientExtensions.cs

```diff
@@ -14,23 +14,24 @@ public static class DebugAdapterClientExtensions
     {
         public static async Task LaunchScript(this DebugAdapterClient debugAdapterClient, string filePath, TaskCompletionSource<object> started)
         {
-            LaunchResponse launchResponse = await debugAdapterClient.Launch(new PsesLaunchRequestArguments
-            {
-                NoDebug = false,
-                Script = filePath,
-                Cwd = "",
-                CreateTemporaryIntegratedConsole = false,
-            }).ConfigureAwait(false);
+            LaunchResponse launchResponse = await debugAdapterClient.Launch(
+                new PsesLaunchRequestArguments
+                {
+                    NoDebug = false,
+                    Script = filePath,
+                    Cwd = "",
+                    CreateTemporaryIntegratedConsole = false
+                }).ConfigureAwait(true);
 
-            if (launchResponse == null)
+            if (launchResponse is null)
             {
                 throw new Exception("Launch response was null.");
             }
 
             // This will check to see if we received the Initialized event from the server.
             await Task.Run(
-                async () => await started.Task.ConfigureAwait(false),
-                new CancellationTokenSource(2000).Token).ConfigureAwait(false);
+                async () => await started.Task.ConfigureAwait(true),
```

Comment:
since you're changing the behavior of the ConfigureAwait consider a comment to explain why the advice of https://docs.microsoft.com/en-us/dotnet/api/system.threading.tasks.task.configureawait?view=net-6.0#remarks isn't applicable.